src/bun.js/bindings/JSBuffer.cpp

```diff
@@ -990,39 +960,44 @@ static inline JSC::EncodedJSValue jsBufferPrototypeFunction_compareBody(JSC::JSG
     switch (callFrame->argumentCount()) {
     default:
```

Comment:
does this always go to the `default` value?

Reply:
only then there's 5 or more arguments